GetFailedUserDataProcessing/handler.ts

```diff
@@ -62,6 +62,9 @@ export const GetFailedUserDataProcessingHandler = (
             failedUserDataProcessingTable,
             choice,
             fiscalCode,
+            // TODO: Refactor for using the new `@azure/data-tables` library
+            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+            // @ts-ignore
             null,
```

Comment:
This code works because the type definition of the library is wrong or incomplete?